### ISSUE TITLE
 AAPT: error: resource android:attr/lStar not found.

### ISSUE THREAD

**Issue by cunha-hudson** [Yukams/background_locator_fixed#134]
when i try build apk i receive this error:
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':background_locator_2:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:C:\Users\Hudson Cunha\Documents\projetos\wss-app-porto-sem-parar\build\background_locator_2\intermediates\merged_res\release\values\values.xml:3664: AAPT: error: resource android:attr/lStar not found.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

Somebody can i help me?
    flutter version  3.24.0 ,  
    compileSdkVersion   = 34
    targetSdkVersion    = 34
    appCompatVersion    = "1.7.0"
    kotlin_version = "2.0.20"

**Comment by darul9061** [comment]
Yes this issue persists.

**Comment by DmitriySimonov** [comment]
same problem

```
* What went wrong:
Execution failed for task ':background_locator_2:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:/home/dmitriysi/repo/dropcat/build/background_locator_2/intermediates/merged_res/release/values/values.xml:3664: AAPT: error: resource android:attr/lStar not found.
```

**Comment by TechBoyy6** [comment]
hey @cunha-hudson  have you found any fix/hack, not able to build and proceed

**Comment by cunha-hudson** [comment]
@TechBoyy6 no, but I realized that the error occurs because the flutter's latest version is incompatible, try use Flutter 3.22.3

**Comment by DmitriySimonov** [comment]
Is there any solution to use this library and latest version of flutter?

**Comment by Alvish0407** [comment]
try adding the code block highlighted below to the `android/build.gradle` file in your project directory.

```diff
rootProject.buildDir = "../build"
subprojects {
    project.buildDir = "${rootProject.buildDir}/${project.name}"
+    afterEvaluate {
+        android {
+            compileSdkVersion 34
+        }
+    }
}
```

**Comment by DmitriySimonov** [comment]
**it didn't help**

Running Gradle task 'assembleRelease'...                        
e: ...pub-cache/hosted/pub.dev/background_locator_2-2.0.6/android/src/main/kotlin/yukams/app/background_locator_2/provider/LocationParserUtil.kt:21:20 Return type mismatch: expected 'java.util.HashMap<kotlin.Any, kotlin.Any>', actual 'java.util.HashMap<kotlin.String, it(kotlin.Comparable<*>? & java.io.Serializable?)>'.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':background_locator_2:compileReleaseKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.


> try adding the code block highlighted below to the `android/build.gradle` file in your project directory.
> 
> ```diff
> rootProject.buildDir = "../build"
> subprojects {
>     project.buildDir = "${rootProject.buildDir}/${project.name}"
> +    afterEvaluate {
> +        android {
> +            compileSdkVersion 34
> +        }
> +    }
> }
> ```

**Comment by 53huzefa-zeenwala** [comment]
Facing Same Issue, Found any Solution?

**Comment by xSILENCEx** [comment]
I tried the fix and updated the example and now it works fine:
```yaml
  background_locator_2:
    git:
      url: https://github.com/xSILENCEx/background_locator_fixed
      ref: master
```
